rust-symcrypt/src/hash.rs

```diff
@@ -110,6 +110,7 @@ pub enum HashAlgorithm {
     Sha3_512,
 }
 
+#[allow(clippy::wrong_self_convention)]
```

Comment:
>clippy::wrong_self_convention) [](http://example.com/codeflow?start=8&length=30)

Add comment for why you added this clippy

Reply:
Renamed to `get_symcrypt_hash`, this was initially flagged by clippy becuase `to_*` in rust usally means that something is being consumed and ownership is taken. `get_symcrypt_hash` and `get_oid_list` is more inline with what this function actually does, and will not trigger clippy. 

This is a `pub(crate)` change and will not be a breaking change.